src/js/swarmAnimation.js

```diff
@@ -7,18 +7,28 @@
  * functions for applying css classes, doing transforms consistently
  */
 
-const swarmAnimation = {
-	hide(el) {
-		el.classList.add('anim-fade--out');
-	},
+const DURATION_TYPE = {
+	short: 'short',
+	medium: 'medium',
+	long: 'long'
+};
 
-	show(el) {
-		el.classList.remove('anim-fade--in');
+const swarmAnimation = {
```

Comment:
For usage in consumer apps, we'll probably just conditionally apply classes in the render method, right?